Makefile

```diff
@@ -68,7 +93,19 @@ finch-core:
 		"$(MAKE)"
 
 	mkdir -p _output
-	cd deps/finch-core/_output && tar c * | tar Cvx  $(OUTDIR)
+	cd deps/finch-core/_output && tar -cf - * | tar -xvf - -C $(OUTDIR)
+	rm -rf $(OUTDIR)/lima-template
+
+.PHONY: finch-core-local
+finch-core-local:
```

Comment:
TODO on removing this rule. IIRC we added this to make lima.exe and the guestagent sourcing the WSL PR to lima: 

* https://github.com/vsiravar/finch-core-public/blob/windev/Makefile#L87
* https://github.com/vsiravar/finch-core-public/blob/windev/Makefile#L152-L156

But once lima is released with WSL support, we should source similar to how we do on macOS via LIMA_URL: https://github.com/vsiravar/finch-core-public/blob/b82c007d22d4d99f8a78a3db7c6252ce9c5afcc8/Makefile#L36C14-L36C14